australian-guide-to-legal-citation.csl

```diff
@@ -199,11 +196,10 @@
         <text variable="title" font-style="italic" text-case="title" form="short"/>
       </if>
       <else>
-        <text variable="title" quotes="true" text-case="title" form="short"/>
+        <text variable="title" form="short" text-case="title" quotes="false"/>
```

Comment:
no, you're misunderstanding my point: The line you're changing also affects subsequent citations for journal articles (and other item types) -- those should still be in quotes. (See the note 62 example in 1.4.2). You need to add a separate else-if for cases (and whichever else item type this applies to).